Riot/Modules/HealthServices/Data/Role.swift

```diff
@@ -22,22 +22,60 @@ class Role: Equatable {
     }
     
     let Name: String
+    let Title: String
     let Identifier: String
-    let Description: String
+    let OfficialName: String
+    
+    //Category, Org Unit, Location and Designation should in reality not be strings but instead links to other objects
     let Designation: String
+    
+    let Location: String
+    //is "Category" actually "Specialty"?
+    let Category: String
+    let OrgUnit: String
+    
     init(withName n: String, andId id: String, andDescription desc: String) {
         Name = n
+        Title = n
```

Comment:
To centralise code can this init call the other init method e.g.
init(n, id, desc, n)

Reply:
Updated these initialisers to be convenience init methods, used them to call the original init method.